upgrade/upgrade-instructions-for-improved-data-fixtures-and-tests.md

```diff
@@ -0,0 +1,63 @@
+# Upgrade Instructions for Improved Data Fixtures and Tests
+
+In [#1425](https://github.com/shopsys/shopsys/pull/1425), we changed the way our data fixtures are created so they are able to adapt to any domains and locales configuration.
+Along with that, we also modified tests to make them resistant against such changes.
+
+Most of the changes were done in `project-base` so if you want them in your project after the upgrade, you need to do that manually.
+If any of the following instructions is not clear enough for you or you need more information, we recommend to take a look at the actual changes in [#1425](https://github.com/shopsys/shopsys/pull/1425).
```

Comment:
Maybe we could create a diff with all the things that are changed by this PR in project-base co upgrading could be faster.

Reply:
Github creates diff for us and in this case it would be just big file same as on Github + we were asking for feedback about upgrading and people say they prefer to look on Github for diffs.